src/serializer/ResidualHeapVisitor.js

```diff
@@ -140,7 +145,9 @@ export class ResidualHeapVisitor {
     }
 
     // prototype
-    this.visitObjectPrototype(obj);
+    if (kind !== "ReactElement") {
```

Comment:
Same thing. Why isn't it ok to visit the prototype? It'll just be Object.prototype which is intrinsic anyway.

Reply:
Same as above